### ISSUE TITLE
Error 200: SyntaxError: JSON.parse: unexpected character

### ISSUE THREAD

**Issue by tsvetkovpro** [netcan/Laravel_AJAX_CRUD#1]
good day, i clone repo and : 
1. composer install
2. php artisan key:generate

run app and when i add task i have an error: 

![image](https://cloud.githubusercontent.com/assets/2004857/18033297/c711a5ca-6d39-11e6-88e5-38247dc82736.png)

**Comment by netcan** [comment]
what is the laravel version?

**Comment by netcan** [comment]
and can you edit the task or delete it?

**Comment by tsvetkovpro** [comment]
i trying to create task and got error. laravel version same as your. i clone your repo and not install laravel myself

**Comment by netcan** [comment]
I have just tested it using following steps:
1. I cloned it and then did `composer install`
2. I copied file `.env.examples` to `.env`, and changed the coulmn `DB_DATABASE` to `DB_DATABASE=task`
3. I created the database `task` in my own MySQL server
4. I ran the `php artisan key:generate` and `php artisan migrate` commands.
5. Lastly, I opened it in Chrome, but I didn't see the problem you described
so I think maybe it's your browser's problem, Can you have a try with Chrome to see whether the problem  resolves?

**Comment by netcan** [comment]
I have just tested it in my roommate's Firefox browser, and found the same problem.

**Comment by tsvetkovpro** [comment]
test in latest chrome with your steps, but same error/

![image](https://cloud.githubusercontent.com/assets/2004857/18094655/ae91033c-6eed-11e6-98ad-2ff3e7874a5a.png)

![image](https://cloud.githubusercontent.com/assets/2004857/18094700/c9bb33f8-6eed-11e6-893d-5eea2f8237a2.png)

what i do wrong?

**Comment by tsvetkovpro** [comment]
my .env file^ 

```
APP_ENV=local
APP_KEY=base64:sYR7oVSU/YUZof9pBrTO78VUxZ3GqP3QiF/Btbpms0w=
APP_DEBUG=true
APP_LOG_LEVEL=debug
APP_URL=http://laravel-ajax-crud/task

DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=task
DB_USERNAME=root
DB_PASSWORD=

CACHE_DRIVER=file
SESSION_DRIVER=file
QUEUE_DRIVER=sync

REDIS_HOST=127.0.0.1
REDIS_PASSWORD=null
REDIS_PORT=6379

MAIL_DRIVER=smtp
MAIL_HOST=mailtrap.io
MAIL_PORT=2525
MAIL_USERNAME=null
MAIL_PASSWORD=null
MAIL_ENCRYPTION=null
```

**Comment by netcan** [comment]
I think you can open Chrome's develop tool(Ctrl + shift + I) and then switch to  `console` tag, post the error information here

**Comment by tsvetkovpro** [comment]
here console input: 

![image](https://cloud.githubusercontent.com/assets/2004857/18094979/e7b579b2-6eee-11e6-9126-e463dab08268.png)

**Comment by netcan** [comment]
Dose my demo `http://115.159.147.250/task` have same problems?

**Comment by tsvetkovpro** [comment]
no man, here not http://115.159.147.250/task problem : help plz !!!!!!!!!!!! what i do wrong?

**Comment by tsvetkovpro** [comment]
i ^ 
1. I cloned it and then did composer install
2. I copied file .env.examples to .env, and changed the coulmn DB_DATABASE to DB_DATABASE=task
3. I created the database task in my own MySQL server
4. I ran the php artisan key:generate and php artisan migrate commands.

**Comment by netcan** [comment]
plz wait, i am adding some `debug` information

**Comment by netcan** [comment]
I have just added some debug information, you can `git pull` and try again.

**Comment by netcan** [comment]
I advise you change domain name and try again.

**Comment by tsvetkovpro** [comment]
thanx a lot, i clone your repo, and : 
1. I cloned it and then did composer install
2. I copied file .env.examples to .env, and changed the coulmn DB_DATABASE to DB_DATABASE=task
3. I created the database task in my own MySQL server
4. I ran the php artisan key:generate and php artisan migrate commands.
5. Lastly, I change domain name

it works nice ! 

![image](https://cloud.githubusercontent.com/assets/2004857/18163882/908514e0-7056-11e6-971c-dd296ec30ae5.png)
